Publish Microsoft Authored templates to azure samples browser (#62)
Azure-Samples/chatgpt-quickstart: README.md

---
page_type: sample
languages:
- azdeveloper
- python
- bicep
- html
products:
- azure
- azure-container-apps
- azure-openai-service
- azure-container-registry
urlFragment: chatgpt-quickstart
name: Simple Chat Application using Azure OpenAI (Python)
description: A sample chat app that uses Python and the Quart framework to stream chat completions from the OpenAI SDK. Easily deployable to Azure Container Apps.
---
<!-- YAML front-matter schema: https://review.learn.microsoft.com/en-us/help/contribute/samples/process/onboarding?branch=main#supported-metadata-fields-for-readmemd -->

> # Build your own ChatGPT app

Reviewer: ```suggestion
# Simple Chat Application using Azure OpenAI (Python)
```

---
page_type: sample
languages:
- azdeveloper
- python
- bicep
- html
products:
- azure
- azure-container-apps
- azure-openai-service
- azure-container-registry

> - azure-pipelines

Reviewer: ```suggestion
```

urlFragment: chatgpt-quickstart
name: Build your own ChatGPT app - Simple Chat Application using Azure OpenAI (Python)

> description: A sample chat app that uses Python and the Flask framework to stream ChatGPT completions as server-sent events to a simple frontend.

Reviewer: ```suggestion
description: A sample chat app that uses Python and the Quart framework to stream chat completions from the OpenAI SDK. Easily deployable to Azure Container Apps.
```
